test/docker-e2e/e2e_test.go

```diff
@@ -53,30 +53,41 @@ func (s *CelestiaTestSuite) CreateCelestiaChainProvider(appVersion string) celes
 		Logger:          s.logger,
 		DockerClient:    s.client,
 		DockerNetworkID: s.network,
-		Type:            "cosmos",
-		Name:            "celestia",
-		Version:         getCelestiaTag(),
-		NumValidators:   &numValidators,
-		NumFullNodes:    &numFullNodes,
-		ChainID:         "celestia",
-		Images: []celestiadockertypes.DockerImage{
-			{
-				Repository: multiplexerImage,
-				Version:    getCelestiaTag(),
-				UIDGID:     "10001:10001",
+		ChainConfig: &celestiadockertypes.ChainConfig{
+			Type:          "cosmos",
+			Name:          "celestia",
+			Version:       getCelestiaTag(),
+			NumValidators: &numValidators,
+			NumFullNodes:  &numFullNodes,
+			ChainID:       "celestia",
+			Images: []celestiadockertypes.DockerImage{
+				{
+					Repository: multiplexerImage,
+					Version:    getCelestiaTag(),
+					UIDGID:     "10001:10001",
+				},
 			},
+			Bin:           "celestia-appd",
+			Bech32Prefix:  "celestia",
+			Denom:         "utia",
+			CoinType:      "118",
+			GasPrices:     "0.025utia",
+			GasAdjustment: 1.3,
+			ModifyGenesis: func(config celestiadockertypes.Config, bytes []byte) ([]byte, error) {
+				return maps.SetField(bytes, "consensus.params.version.app", appVersion)
+			},
+			EncodingConfig:      &enc,
+			AdditionalStartArgs: []string{"--force-no-bbr", "--grpc.enable", "--grpc.address", "0.0.0.0:9090", "--rpc.grpc_laddr=tcp://0.0.0.0:9099"},
```

Comment:
[nit] does the RCP GRPC listen address need to differ from the default `9098`? https://github.com/celestiaorg/celestia-app/blob/6a2e7513dbb2c11808c1e8c99d35532d4d946aa1/app/default_overrides.go#L272

```suggestion
			AdditionalStartArgs: []string{"--force-no-bbr", "--grpc.enable", "--grpc.address", "0.0.0.0:9090", "--rpc.grpc_laddr=tcp://0.0.0.0:9098"},
```

Reply:
it shouldn't actually matter which port is used here, but no harm being consistent with the default.